haskell/src/Lorentz/Contracts/Spec/FA2Interface.hs

```diff
@@ -165,8 +165,12 @@ instance Arbitrary OwnerTransferMode where
 instance TypeHasDoc OwnerTransferMode where
   typeDocMdDescription = "Describes if owener hooks are required in sender/receiver addresses"
 
-type CustomPermissionPolicy
-  = ("tag" :! MText, "config_api" :! Maybe Address)
+data CustomPermissionPolicy = CustomPermissionPolicy
```

Comment:
Hm, apparently this type is unused. I think I'll create a new issue about finding unused types are removing them.

Reply:
https://github.com/tqtezos/stablecoin/issues/115